_summaries/graham2021levit_1.md

```diff
@@ -1,19 +1,27 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: graham2021levitt
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: biofizzatreya
+score: 8
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+The paper attempts to combine, convolutional networks and training principle from CNNs with vision transformers.
+![image](https://user-images.githubusercontent.com/13065170/136635751-c21a6e5d-c6ce-4813-bda9-4f3ef884e265.png)
+![image](https://user-images.githubusercontent.com/13065170/136635765-4209a6b7-7e90-436c-a96f-2cb958586416.png)
+
+The transformer layers are stacked like a ResNet. The ResNet extracts features from the transformer layers and the gradients are back-propagated. Thus all CNN operations can be applied on the intermediate representations.
+![image](https://user-images.githubusercontent.com/13065170/136635772-1c66741e-e1f9-4927-af8c-df03e3d3ad3b.png)
+![image](https://user-images.githubusercontent.com/13065170/136635778-6ab9bdbe-7889-4779-b60a-204815006999.png)
+
+LeVit is 1.5 to 5 times faster than competition. 
+![image](https://user-images.githubusercontent.com/13065170/136635784-094d7434-e487-40a4-b3fa-a14a7eb44a4e.png)
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* CNN strategies applied to vision transformers
+* Convolutional operations applied to transformer stacks
+* Results in much faster object detection
```

Comment:
Is it object detection or merely image classification? Is the final 4x4 feature map sufficient for object detection?